### ISSUE TITLE
Exhibit card background color missing

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/exhibits#2860]
The background color is missing from the exhibit cards? This may create a contrast issue when the card text is expanded upon hover and overlays the card image.

### Current implementation
<img width="1489" alt="Image" src="https://github.com/user-attachments/assets/383342fb-55bb-4e26-89a2-df925a64262d" />

### Previous implementation
![Image](https://github.com/user-attachments/assets/f2accc30-0258-4039-827f-7ab0f77a6aa4)

**Comment by corylown** [comment]
@alundgard is there a specification somewhere for what the color and opacity of the card should be? There is a color and opacity set, but we may need to set it to something else to achieve the desired effect.

**Comment by alundgard** [comment]
@corylown Yes the previous implementation was #2654. Background color: **Fog Light** `#F4F4F4` Opacity: 90%.
